src/construction/all_data_construction.py

```diff
@@ -20,7 +19,7 @@
 )
 
 
-def all_data_construction(
+def all_data_construction(  # noqa: C901
```

Comment:
what is this noqa: C901 commented out?

Reply:
it's the way you do a "no QA" along with the code